### PR TITLE
[5.2] Create a csrf_json_token

### PR DESCRIPTION
**Pull request by socieboy**
I think it's a clean way to have the csrf token attached on the script! that's it, but it's only the way i think!

### CODE REVIEW

**Comment by socieboy** [comment]
First pull request sorry about the changes...

**Comment by RomainLanz** [comment]
`window.Laravel` cannot be changed with your commits because you are hard-coding the `json_encode` method.

File: src/Illuminate/Foundation/helpers.php [hunk 1]
```diff
@@ -832,3 +832,15 @@ function view($view = null, $data = [], $mergeData = [])
         return $factory->make($view, $data, $mergeData);
     }
 }
+
+if (! function_exists('csrf_json_token')) {
+    /**
+     * CSRF Token to json.
+     *
+     * @return \Illuminate\Support\HtmlString
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/helpers.php]: This doesn't return an html string instance.

File: src/Illuminate/Auth/Console/stubs/make/views/layouts/app.stub [hunk 2]
```diff
@@ -14,11 +14,7 @@
     <link href="/css/app.css" rel="stylesheet">
 
     <!-- Scripts -->
-    <script>
-        window.Laravel = <?php echo json_encode([
-            'csrfToken' => csrf_token(),
-        ]); ?>
-    </script>
+    <script>window.Laravel = {!! csrf_json_token() !!}</script>
```
Comment (GrahamCampbell) [inline src/Illuminate/Auth/Console/stubs/make/views/layouts/app.stub]: {{ }} would be fine here.

**Comment by taylorotwell** [comment]
I don't want to do this because I want it to be simple and easy to add additional values to the Laravel object.

**Pull request closed without merging**